DonjinKrawler-Common/src/main/java/krawlercommon/RegistrationManager.java

```diff
@@ -0,0 +1,55 @@
+package krawlercommon;
+
+import com.esotericsoftware.kryo.Kryo;
+import krawlercommon.enemies.Chicken;
+import krawlercommon.enemies.Enemy;
+import krawlercommon.enemies.Skeleton;
+import krawlercommon.enemies.Zombie;
+import krawlercommon.enemies.big.BigChicken;
+import krawlercommon.enemies.big.BigSkeleton;
+import krawlercommon.enemies.big.BigZombie;
+import krawlercommon.enemies.small.SmallChicken;
+import krawlercommon.enemies.small.SmallSkeleton;
+import krawlercommon.enemies.small.SmallZombie;
+import krawlercommon.packets.*;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class RegistrationManager {
+
+    public static void registerKryo(Kryo kryo) {
```

Comment:
Čia kaip suprantu šitoje klasėje tą biblioteką užregistruoja?

Reply:
KryoNet'as po apačia naudoja Kryo serializacijai, Kryo reikalauja, kad visos serializuojamos klasės būtų "užregistruotos". Tuo pačiu KryoNet'as reikalauja, kad tos klasės būtų užregistruotos ir Client'e ir Serveryje, ir būtinai vienoda tvarka. Šita klasė ir šis metodas - centralizuota vieta laikyt visas registracijas.